### PR TITLE
Eb 2854 branch

### PR DESCRIPTION
**Pull request by jakeiotechsys**

### CODE REVIEW

File: edgebuilder-install.sh [hunk 1]
```diff
@@ -297,7 +302,7 @@ install_node()
     echo "WARN: docker.socket is enabled, disabling..."
     systemctl disable docker.socket
   fi
-  for i in docker docker-engine docker.io containerd runc; do
+  for i in docker docker-engine docker.io containerd runc docker-ce docker-ce-cli docker-compose-plugin; do
```
Comment (rbose21) [inline edgebuilder-install.sh:305]: Could you please add "docker-ce-rootless-extras" to this list? Thanks

File: edgebuilder-install.sh [hunk 2]
```diff
@@ -511,9 +516,8 @@ if [ "$COMPONENT" = "server" ];then
     exit 1
   fi
 elif [ "$COMPONENT" = "node" ]; then
-
   if [ "$ARCH" = "x86_64" ]||[ "$ARCH" = "aarch64" ];then
-    if [ "$OS" = "$UBUNTU2204" ]||[ "$OS" = "$UBUNTU2004" ]||[ "$OS" = "$UBUNTU1804" ]||[ "$OS" = "$DEBIAN11" ]||[ "$OS" = "$DEBIAN10" ];then
+    if [ "$OS" = "$UBUNTU2204" ]||[ "$OS" = "$UBUNTU2004" ]||[ "$OS" = "$UBUNTU1804" ]||[ "$OS" = "$DEBIAN11" ];then
```
Comment (rbose21) [inline edgebuilder-install.sh:520]: We definitely need to support Debian10 x85_64

Comment (jakeiotechsys) [reply edgebuilder-install.sh:527]: I found that deb 10 wasn't supported for arm64, so I removed it. But yes obviously it needs to be added to x86. I've added a separate section for arm64 compatibility checking.

File: edgebuilder-install.sh (hunk 1 above)
Comment (jakeiotechsys) [reply edgebuilder-install.sh]: When I tested the installer I didn't see that this was installed. But that's fine I've added it

File: edgebuilder-install.sh [hunk 3]
```diff
@@ -511,14 +516,20 @@ if [ "$COMPONENT" = "server" ];then
...
-
-  if [ "$ARCH" = "x86_64" ]||[ "$ARCH" = "aarch64" ];then
+  if [ "$ARCH" = "x86_64" ];then
     if [ "$OS" = "$UBUNTU2204" ]||[ "$OS" = "$UBUNTU2004" ]||[ "$OS" = "$UBUNTU1804" ]||[ "$OS" = "$DEBIAN11" ]||[ "$OS" = "$DEBIAN10" ];then
       install_node "$OS" "$ARCH"
     else
       echo "ERROR: The Edge Builder node components are not supported on $OS - $ARCH"
       exit 1
     fi
+  elif [ "$ARCH" = "aarch64" ];then
```
Comment (rbose21) [inline edgebuilder-install.sh:526]: Jake, one more qn: could the machine/CPU arch be "arm64" as well?

Comment (jakeiotechsys) [reply edgebuilder-install.sh:526]: I haven't seen it in my testing. I think arm64 is just shorthand for aarch64

Comment (jakeiotechsys) [reply edgebuilder-install.sh:526]: Actually, aarch64 is the formal name for it apparently

**Review by rbose21 — APPROVED** [review]
LGTM

**Pull request merged**